engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/FramePointer.java

```diff
@@ -8,8 +9,9 @@
 /**
  * A representation of a pointer into a stack frame at a given number of levels above the current.
  */
-@Persistable(clazz = FramePointer.class, id = 1283)
-public record FramePointer(int parentLevel, int frameSlotIdx) implements FrameAnalysisMeta {
+@Persistable(clazz = FramePointer.class, id = 1288)
+public record FramePointer(int parentLevel, int frameSlotIdx, UUID externalId)
```

Comment:
- I'd prefer `FramePointer` to remain simple pair of numbers
  - it gets serialized into the caches - it should be small
- What's the rationale for associating `externalId` with a frame pointer?
   - is it the definition ID or usage ID?